platform/mellanox/mlnx-platform-api/sonic_platform/platform.py

```diff
@@ -15,11 +16,14 @@ class Platform(PlatformBase):
     def __init__(self):
         PlatformBase.__init__(self)
         self._chassis = Chassis()
-        self._chassis.initialize_psu()
         self._chassis.initialize_eeprom()
-        if utils.is_host():
-            self._chassis.initialize_components()
-            self._chassis.initizalize_system_led()
-        else:
-            self._chassis.initialize_fan()
-            self._chassis.initialize_thermals()
+        platform_name = get_platform()
+        simx_platform = bool(platform_name and "simx" in platform_name)
```

Comment:
@tomer-israel why do we need `bool()`?

Reply:
I changed it, now it checks that "simx" not exist on platform_name